acceptance/config/beaker/options.rb

```diff
@@ -6,6 +6,7 @@
   "service-prefix" => 'service ',
   "service-num-retries" => 1500,
   "puppetservice" => 'puppetserver',
```

Comment:
I take it that during PE testing the PE version of this `options.rb` will be used, right?  Otherwise the service name wouldn't match (`pe-puppetserver`)